SportsHub/SportsHub.Application/Services/CommentService.cs

```diff
@@ -16,5 +17,28 @@ public async Task<IEnumerable<Comment>> GetByArticleAsync(int id)
         {
             return await _unitOfWork.CommentRepository.GetByArticleAsync(id);
         }
+
+        public async Task<bool> PostCommentAsync(PostCommentDTO commentInput)
+        {
+            var userExists = _unitOfWork.UserRepository.GetById(commentInput.AuthorId) != null;
+            var articleExists = _unitOfWork.ArticleRepository.GetById(commentInput.ArticleId) != null;
+
+            if (!userExists || !articleExists)
+            {
+                return false;
+            }
+
+            var comment = new Comment()
+            {
+                Content = commentInput.Content,
+                AuthorId = commentInput.AuthorId,
+                ArticleId = commentInput.ArticleId,
+            };
+
+            await _unitOfWork.CommentRepository.PostCommentAsync(comment);
```

Comment:
Please do not use HTTP verbs in name of repository method.

Reply:
Fixed.